code/API_definitions/checkSimSwap-v0.3.0.camara.swagger.yaml

```
@@ -0,0 +1,288 @@
+                    code: INVALID_INPUT
+                    message: "Expected property is missing: msisdn"
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "409":
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorInfo"
+              example:
+                code: CONFLICT
+                message: "Another resquest is created for the same UE"
+        "500":
+          description: Server error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorInfo"
+              example:
+                code: INTERNAL
+                message: "Session could not be created"
+        "503":
+          $ref: "#/components/responses/Generic503" 
+  /checkSimSwap:
+    post:
+      tags:
+      - Check simSwap
+      description: Manage request and response for a simSwap check for a mobile user account provided with MSIDN.
+      operationId: createCheckSimSwap
+      requestBody:
+        description: |+
+          Create a Check simSwap request. A MSISDN identifier and age are required.        
+        content :
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateCheckSimSwap"
+        required: true
+      responses:
+        "201":
+          description: check simSwap created
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/CheckSimSwapInfo"
+        "400":
+          description: Invalid input
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorInfo"
+              examples:
+                Generic400:
+                  summary: Schema validation failed
+                  value: 
+                    code: INVALID_INPUT
+                    message: "Schema validation failed at  ..."
+                MsisdnRequired:
+                  summary: msisdn is required as part of ueId
+                  value: 
+                    code: INVALID_INPUT
+                    message: "Expected property is missing: msisdn"
+                AgeRequired:
+                  summary: msisdn is required as part of ueId
+                  value: 
+                    code: INVALID_INPUT
+                    message: "Expected property is missing: age"                    
+        "401":
+          $ref: "#/components/responses/Generic401"
+        "403":
+          $ref: "#/components/responses/Generic403"
+        "409":
+          description: Conflict
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorInfo"
+              example:
+                code: CONFLICT
+                message: "Another resquest is created for the same UE"

```

```suggestion
                message: "Another resquest is created for the same MSISDN"
```

Again, noty sure if we should consider this